clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -3849,6 +3849,30 @@ LogicalResult cir::CatchParamOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// StdInitializerListOp Definitions
+//===----------------------------------------------------------------------===//
+
+LogicalResult cir::StdInitializerListOp::verify() {
```

Comment:
We also require tests to cover custom verification. See `clang/test/CIR/IR/invalid.cir`.